bug: Tfsec fails during parsing

**Describe the bug**
Since 1.26.1 release, I encounter an exception while tfsec is parsing my module:
```
panic: not a number
```

**To Reproduce**
Steps to reproduce the behavior:
1. I just execute tfsec on my project and get the error above

**Expected behavior**
No panic error (or maybe a more explicit error)

**Output of your tfsec command with --debug flag**
```
10:02.298695299 system.info                      APP       tfsec
10:02.298732227 system.info                      VERSION   v1.26.2
10:02.298733661 system.info                      OS        linux
10:02.298734864 system.info                      ARCH      amd64
10:02.298736463 system.info                      KERNEL    Linux version 5.17.15-76051715-generic (jenkins@warp.pop-os.org) (gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0, GNU ld (GNU Binutils for Ubuntu) 2.38)
10:02.298739617 system.info                      TERM      xterm-256color
10:02.298741255 system.info                      SHELL     /bin/bash
10:02.298742600 system.info                      GOVERSION go1.18.3
10:02.298745793 system.info                      GOROOT    /opt/hostedtoolcache/go/1.18.3/x64
10:02.298747549 system.info                      CGO       false
10:02.298749083 system.info                      CPUCOUNT  12
10:02.298750381 system.info                      MAXPROCS  12
10:02.298751511 system.info                      WORKDIR   [...]
10:02.298755114 system.info                      UID       1000
10:02.298756639 system.info                      EUID      1000
10:02.298757768 system.info                      DOCKER    false
10:02.298759395 system.info                      CI        false
10:02.298760626 system.info                      HOSTNAME  [...]
10:02.298762930 system.info                      TEMP      /tmp
10:02.298766957 system.info                      PATHSEP   /
10:02.298768410 system.info                      CMD       tfsec --debug
10:02.298771113 cmd                              Command args=[]string{}
10:02.298794442 cmd                              Determined path dir=[...]
WARNING: A tfvars file was found but not automatically used. Did you mean to specify the --tfvars-file flag?
10:02.298894491 cmd                              Determined path root=/
10:02.298896215 cmd                              Determined path rel=[...]
10:02.298922466 terraform.scanner                Scanning [&{/ /}] at [...]...
10:02.299667852 terraform.scanner.rego           Loaded 4 embedded libraries.
10:02.318132120 terraform.scanner.rego           Loaded 72 embedded policies.
10:02.360842921 terraform.scanner                Scanning root module [...]...
10:02.360864636 terraform.parser.<root>          Setting project/module root to [...]
10:02.360875879 terraform.parser.<root>          Parsing FS from [...]
10:02.360981727 terraform.parser.<root>          Parsing [...]...
10:02.361245329 terraform.parser.<root>          Added file [...].
10:02.396809800 terraform.parser.<root>          Evaluating module...
10:02.421489046 terraform.parser.<root>          Read 482 block(s) and 0 ignore(s) for module 'root' (32 file[s])...
10:02.421514126 terraform.parser.<root>          Added 0 variables from tfvars.
10:02.421970832 terraform.parser.<root>          Loaded module metadata for 311 module(s) from '[...].terraform/modules/modules.json'.
10:02.421982579 terraform.parser.<root>          Working directory for module evaluation is [...]
10:02.422116027 terraform.parser.<root>.evaluator Filesystem key is [...]
10:02.422119541 terraform.parser.<root>.evaluator Starting module evaluation...
10:02.429630497 terraform.parser.<root>.evaluator Expanded block [...] into 1 clones via 'count' attribute.
10:02.431200657 terraform.parser.<root>.evaluator Starting submodule evaluation...
10:02.431221307 terraform.parser.<root>.evaluator Module [...] resolved to path [...] in filesystem '&{/ /}' using modules.json
10:02.433166512 terraform.parser.<root>.evaluator found module [...] in .terraform/modules
10:02.433171812 terraform.parser.<root>.evaluator Loaded module [...] from [...].
10:04.313226557 terraform.parser.<root>.evaluator Finished processing 237 submodule(s).
10:04.313239994 terraform.parser.<root>.evaluator Starting post-submodule evaluation...
10:04.320514075 terraform.parser.<root>.evaluator Module evaluation complete.
10:04.320550523 terraform.parser.<root>          Finished parsing module 'root'.
10:04.320557990 terraform.parser.<root>.evaluator Added module output [...]
10:04.320707813 terraform.executor               Adapting modules...
panic: not a number
```

**System Info**
<!-- complete the following -->
 - tfsec version: `1.26.2`
 - terraform version: `1.1.7`
 - OS: `Pop!_OS 22.04 LTS`

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/Masterminds/semver v1.5.0
-	github.com/aquasecurity/defsec v0.68.7
+	github.com/aquasecurity/defsec v0.68.8
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-version v1.5.0
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf

